prometheus_hardware_exporter/collector.py

```diff
@@ -547,14 +548,17 @@ def fetch(self) -> List[Payload]:
                     "Unknown ipmi SEL state: %s. Treating it as Nominal.", sel_entry["State"]
                 )
                 sel_state_value = sel_states_dict["NOMINAL"]
+            sel_entries_tuples.add((sel_entry["Name"], sel_entry["Type"], sel_state_value))
```

Comment:
**question (blocking):** 

If I have two records which have the same name and type, but the state is different: Nominal and Warning.

I will get two metrics? I will suggest to return one record with the urgent state. For this case should be only one Warning record. Multiple records with same name but different state will confuse people.

Reply:
Status will be `Nominal` in most of time. And if the status is `Warning` or `Critical`, user wouldn't care the `Nominal` status. Don't use exporter as logging system. You should provide **single true of source**, which is the most urgent status to user.

Reply:
For lost information. Exporter shouldn't try to provide the information which logging should provide.
And don't provide the duplicate metrics in the exporter. This always confuse people.